styles.css

```diff
@@ -1,826 +1,980 @@
-  width: 12px;
-  color: #acb7c3;
-  opacity: 0.5;
-  float: right;
-  position: relative;
-  right: 20px;
-}
-
-.icon-section {
-  display: flex;
-  align-items: center;
-  letter-spacing: 0.05em;
-  margin-left: 1.5rem;
-  margin-bottom: 1.5rem;
-  margin-top: -10px;
-}
-
-.icon-section span {
-  margin-left: 4rem;
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
-  color: #253858;
-}
-
-li.fram {
-  font-family: Poppins, sans-serif;
-  height: 38px;
-  width: 36px;
-  font-style: normal;
-  font-weight: 100;
-  font-size: 20px;
-  line-height: 24px;
-  margin-top: 1rem;
-  margin-bottom: 0;
-  margin-left: 2rem;
-  color: #000;
-}
-
-li.skills {
-  font-family: Poppins, sans-serif;
-  height: 210px;
-  width: 36px;
-  font-style: normal;
-  font-weight: 100;
-  font-size: 20px;
-  line-height: 24px;
-  margin-top: 1rem;
-  margin-left: 2rem;
-  color: #000;
-}
-
-/* ---------Form Section------------ */
-
-form {
-  display: flex;
-  flex-direction: column;
-}
-
-.contact-form {
-  margin-top: 10rem;
-  background-color: #6070ff;
-  background-repeat: no-repeat;
-  background-position-x: 350px;
-  padding: 80px 10px 10px;
-  border-top-left-radius: 120px;
-  text-align: center;
-}
-
-.contact-content h1 {
-  font-family: Poppins, sans-serif;
-  color: #fff;
-  text-align: center;
-  height: 56px;
-  font-size: 40px;
-  line-height: 56px;
-  margin: 20px 0;
-}
-
-.contact-content p {
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  color: #ebebff;
-  height: 136px;
-  font-size: 20px;
-  line-height: 28px;
-  font-weight: 4000;
-  text-align: center;
-}
-
-.name-container input {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 48px;
-  padding-left: 5px;
-}
-
-.mail-container input {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 48px;
-  padding-left: 5px;
-}
-
-.textarea-container textarea {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 170px;
-  padding-left: 5px;
-}
-
-button {
-  width: 140px;
-  height: 48px;
-  margin-left: 40px;
-  border: 1px solid #6070ff;
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-  background: #fff;
-  border-radius: 8px;
-  color: #6070ff;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
+/* HTML5 Reset Stylesheets. */
+
+/* -----Header Section ---- */
+@media only screen and (min-width: 376px) {
```

Comment:
- Please, kindly use a media query not lower than 768px to meet the project requirement.